iron-selectable.html

```diff
@@ -241,6 +242,12 @@
       this._setItems(nodes);
     },
 
+    _updateAttrForSelected: function() {
+      if (this._shouldUpdateSelection) {
+        this.selected = this._indexToValue(this.indexOf(this.selectedItem));        
```

Comment:
Nice, I was looking at this issue earlier and was trying to do this by caching the old value of `attrForSelected`. This is much cleaner.